robottelo/cli/factory.py

```diff
@@ -3934,20 +3743,23 @@ def setup_cdn_and_custom_repositories(
 
 
 def setup_cdn_and_custom_repos_content(
-        org_id, lce_id, repos, upload_manifest=True,
-        download_policy='on_demand', rh_subscriptions=None):
+        org_id, repos, lce_id=None, upload_manifest=True,
```

Comment:
Please do not change the order of the args of this function as used by many other functions
```
 org_id, lce_id=None, repos=None, upload_manifest=True,
```
and in the body of the function also add 
```
if repos is None:
    repos = []
```

Reply:
This function is not used by capsule setup only

Reply:
thanks for comment, will check this out.